src/server/pachyderm_test.go

```diff
@@ -126,10 +125,10 @@ func TestDuplicatedJob(t *testing.T) {
 		}},
 	}
 
-	job1, err := pachClient.CreateJob(context.Background(), req)
+	job1, err := c.PpsAPIClient.CreateJob(context.Background(), req)
```

Comment:
Calls like this should fall through the the right embedded client, right? (Just like in the `c.CreateRepo(...)` call a few lines above?)

Whatever pattern we expect users to use we should also use in the tests. I think thats the simpler `c.CreateJob(...)`